api/src/main/java/com/mybooks/api/model/Author.java

```diff
@@ -0,0 +1,29 @@
+package com.mybooks.api.model;
+
+import com.sun.istack.NotNull;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Author {
+    @Id
+    @NotNull
+    @GeneratedValue(generator = "system-uuid")
+    @GenericGenerator(name = "system-uuid", strategy = "uuid")
+    private String id;
+
+    @NotNull
+    private String firstName;
+
+    @NotNull
+    private String lastName;
+}
```

Comment:
The Author should have list of books, where is the db relation one to many.
https://www.baeldung.com/hibernate-one-to-many